plugins/modules/net_tools/dnsmadeeasy.py

```diff
@@ -623,7 +623,7 @@ def main():
     # Fetch existing monitor if the A record indicates it should exist and build the new monitor
     current_monitor = dict()
     new_monitor = dict()
-    if current_record and current_record['type'] == 'A':
+    if current_record and current_record['type'] == 'A' and current_record['monitor']:
```

Comment:
Is it potentially better to do this:
```suggestion
    if current_record and current_record['type'] == 'A' and current_record.get('monitor'):
```
so it won't crash when `monitor` is not a valid key in `current_record`?

Reply:
Thanks for the suggestion. It is absolutely better, as far as i experienced, the key should be present, but with this we can avoid issues if this changes